docs/tests_ci.md

```diff
@@ -0,0 +1,217 @@
+## Automation and Tests
+
+Tests are run during continuous integration to catch any errors and to preview
+content. Specifically, usrse.github.io uses the following integrations (with links
+to configuration files):
+
+ - [CircleCI](.circleci/config.yml) previews the site, and tests jobs and mapdata
+ - [GitHub CI](.github/workflows) includes GitHub triggers and actions
+
+Instructions for running locally, along with details about each, are provided below.
+
+## CircleCI
+
+CircleCI is the primary means to preview a pull request, as the site is built and available
+for preview as an artifact. Additionally, the jobs and map data is tested (details below).
+There are no credentials or secrets required for this setup, other than the repository
+needing to be connected to CircleCI, and under settings:
+
+ - build forked pull requests should be on
+ - cancel redundant builds is suggested
+ - workflows should be enabled
+
+If you want to edit any of the tests, you should edit [config.yml](.circleci/config.yml).
+Details about running tests locally are included below. This can be good to do if you
+change an input file in [_data](_data) and want to test it.
+
+### Previewing the Site 
+
+CircleCI builds the site and creates preview of individual pages. 
+The preview only works for individual pages (the links within the preview site don't link to other pages in preview).  
+
+In the PR, you'll see:
+
+![PR checks notification](assets/img/pr_checks.png)
+
+Click on Show All Checks, then get Details for the CircleCI checks:
+
+![PR checks details](assets/img/pr_checks_detailed.png)
+
+This will take you to the CircleCI page, where you should click on the Artifacts link:
+
+![CircleCI project page](assets/img/circleci.png)
+
+This will open a list of the individual pages - click on links to review the pages (reminder: internal links won't go to other preview pages)
+
+![CircleCI preview links](assets/img/circleci_artifacts.png)
+
+### Test Jobs
```

Comment:
"Test jobs" is a bit confusing and hard to parse. This could be testing of job additions, or jobs that test the website. I wonder if another word would help, perhaps "testing additions of new jobs"?

Reply:
I agree with @danielskatz